tests/components/vesync/common.py

```diff
@@ -1,7 +1,30 @@
 """Common methods used across tests for VeSync."""
 import json
 
-from tests.common import load_fixture
+import requests_mock
+
+from homeassistant.components.vesync.const import DOMAIN
+
+from tests.common import load_fixture, load_json_object_fixture
+
+ALL_DEVICES = load_json_object_fixture("vesync-devices.json", DOMAIN)
+ALL_DEVICE_NAMES: list[str] = [
+    dev["deviceName"] for dev in ALL_DEVICES["result"]["list"]
+]
+
```

Comment:
I suggest that you define a constant with the fixture corresponding to each `device_name`
```suggestion
DEVICE_FIXTURES: dict[str, list[tuple[str, str, str]]] = {
    "Humidifier 200s": [
        ("post", "/cloud/v2/deviceManaged/bypassV2", "device-detail.json")
    ],
    "Humidifier 600S": [],
    "Air Purifier 131s": [
        ("post", "/131airPurifier/v1/device/deviceDetail", "purifier-detail.json")
    ],
    "Air Purifier 200s": [
        ("post", "/cloud/v2/deviceManaged/bypassV2", "device-detail.json")
    ],
    "Air Purifier 400s": [
        ("post", "/cloud/v2/deviceManaged/bypassV2", "device-detail.json")
    ],
    "Air Purifier 600s": [
        ("post", "/cloud/v2/deviceManaged/bypassV2", "device-detail.json")
    ],
    "Dimmable Light": [
        ("post", "/SmartBulb/v1/device/devicedetail", "device-detail.json")
    ],
    "Temperature Light": [
        ("post", "/cloud/v1/deviceManaged/bypass", "device-detail.json")
    ],
    "Outlet": [("get", "/v1/device/outlet/detail", "outlet-detail.json")],
    "Wall Switch": [
        ("post", "/inwallswitch/v1/device/devicedetail", "outlet-detail.json")
    ],
    "Dimmer Switch": [("post", "/dimmer/v1/device/devicedetail", "dimmer-detail.json")],
}

```